lib/fpm/package/cpan.rb

```diff
@@ -309,17 +309,16 @@ def download(metadata, cpan_version=nil)
                  :distribution => distribution,
                  :version => cpan_version)
 
-    # default to latest versionunless we specify one
+    # default to latest version unless we specify one
     if cpan_version.nil?
-      self.version = metadata["version"]
+      self.version = "#{metadata["version"]}"
```

Comment:
Gotcha. My preferred way to do this is to call `.to_s` on the object: `metadata["version"].to_s`

No change required. Thanks for the explanation :)